package.json

```
@@ -158,7 +158,7 @@
     "version": "yarn node ./bin/after-version-bump.js",
     "watch:css": "onchange 'packages/{@uppy/,}*/src/*.scss' --initial --verbose -- yarn run build:css",
     "watch:js:bundle": "onchange 'packages/{@uppy/,}*/src/**/*.js' --initial --verbose -- yarn run build:bundle",
-    "watch:js:lib": "onchange 'packages/{@uppy/,}*/src/**/*.js' --initial --verbose -- yarn run build:lib",
+    "watch:js:lib": "onchange 'packages/{@uppy/,}*/src/**/*.{js,ts}' --initial --verbose -- yarn run build:lib",
```

I don't think we have much `.js` files left
```suggestion
    "watch:js:lib": "onchange 'packages/{@uppy/,}*/src/**/*.{ts,tsx}' --initial --verbose -- yarn run build:lib",
```

---

As long as something is there, let's not confuse ourselves when we work with JS and just watch it all.

```suggestion
    "watch:js:lib": "onchange 'packages/{@uppy/,}*/src/**/*.{js,ts,jsx,tsx}' --initial --verbose -- yarn run build:lib",
```

---

Makes sense. Let's take it to a PR that will land on `main`: https://github.com/transloadit/uppy/pull/5046
```suggestion
    "watch:js:lib": "onchange 'packages/{@uppy/,}*/src/**/*.js' --initial --verbose -- yarn run build:lib",
```